definitions/ext-cohesity/dashboard.json

```diff
@@ -0,0 +1,285 @@
+{
+  "name": "Backup - Cohesity",
+  "description": null,
+  "pages": [
+    {
+      "name": "Cohesity",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Data collection (3 clusters)",

```

Comment:
Why 3 clusters? It cannot be higher/lower for other users?

Reply:
It can for sure (3 clusters) - should be removed from the title

Reply:
Removed

Reply:
@otaviocarvalho - updated.

Reply:
@otaviocarvalho - updated. Could it go for a merge?